content/topics/java-specific/concurrency/_index.md

```diff
@@ -0,0 +1,92 @@
+---
+content_type: topic
+ready: true
+tags:
+- threads
+- concurrency
+title: Java Multithread, Concurrency and Parallelism
+---
+
+By now, you should have heard about Java being multi-threaded - but what does that mean exactly?
+
+Unlike JavaScript for example - which is a single threaded language; Java has multiple threads to help you run your program faster. 
+
```

Comment:
If I didn't know what multi-threading was then I would be frustrated and confused for a while longer than needed. 

One trick to editing well is: Removing things and re-ordering things.

```
You may have heard about Java being multi-threaded - but what does that mean exactly?

A thread is like a separate CPU executing your application. Thus, a multithreaded application is like an application that has multiple CPUs executing different parts of the code at the same time. Multi-threaded applications have multiple threads so they can be much faster than single-threaded applications.

A thread goes through a life-cycle which can be seen in the diagram below:

...
```